packages/openneuro-server/src/routes.ts

```diff
@@ -157,6 +163,48 @@ const routes = [
     middleware: [noCache, orcid.authCallback],
     handler: jwt.authSuccessHandler,
   },
+
+  // GitHub authentication route
+  {
+    method: "get",
+    url: "/auth/github",
+    middleware: [storeRedirect],
+    handler: (req, res, next) => {
+      const redirectTo = req.query.redirectTo || "/"
+      passport.authenticate("github", {
+        state: encodeURIComponent(redirectTo),
+      })(req, res, next)
+    },
+  },
+
+  // GitHub OAuth callback route
+  {
+    method: "get",
+    url: "/auth/github/callback",
+    handler: (req, res, next) => {
+      passport.authenticate("github", (err, user) => {
+        if (err) {
+          // eslint-disable-next-line
+          console.error("GitHub Auth Error:", err)
```

Comment:
It would be good to also capture a Sentry error here, especially useful for debugging auth issues in production.

Reply:
added sentry

Reply:
is there a way to test Sentry?